Calendar/Common/EventStoreWrapper.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  EventStoreWrapper.swift
+//  Calendar
+//
+//  Created by Ankush Gupta on 07/08/22.
+//
+
+import EventKit
+
+struct EventStoreWrapper: PermissionInterface {
+  let store: EKEventStore
+  init() {
```

Comment:
Add `event` as a property inside EventStoreWrapper. And Initialize its value based on the init parameter.